src/pages/society/executives.astro

```diff
@@ -34,19 +34,12 @@ EXECUTIVES.forEach((executive) => {
 						<div class="pl-1">
 							{exec.username && (
 								<p class="text-sm">
-									<code>{exec.username}</code>
-									<span class="opacity-70"> at </span>
-									<code>mun</code>
-									<span class="opacity-70"> dot </span>
-									<code>ca</code>
+									<code>{exec.username}</code><span class="opacity-70">@</span><code>mun</code><span class="opacity-70">.</span><code>ca</code>
```

Comment:
i remember now, the reasno it was at dot was so that we don't get automated web scraping spam to our emails

Reply:
Yes but that's ugly :( is there another alternative?